### ISSUE TITLE
Feature request: Transpile react-router-native to ES5

### ISSUE THREAD

**Issue by JulianMayorga** [ReactTraining/react-router#5882]
**TLDR: It would allow apps generated with `create-react-app` to use both `react-native-web` and `react-native-router`**

Transpiling react-router-native to ES5 would allow people to use this package in an app generated with `create-react-app`. Right now using `react-router-native` in a `create-react-app` which also uses `react-native-web` yields the following error (I replaced the default `App.js` code with the one from `react-router-native` [Basic example](https://reacttraining.com/react-router/native/example/Basic)):

![screen shot 2018-01-17 at 21 08 59](https://user-images.githubusercontent.com/843342/35074323-133ad2d8-fbcc-11e7-92bb-219ab0b62c55.png)

**Comment by timdorr** [comment]
I'm fine with throwing a transpiled version in there, but that would seem to be a build tooling configuration issue, right? That's outside the scope of us to address.

**Comment by JulianMayorga** [comment]
You're right that it's more of a build tooling issue, now that you put it this way I totally agree with you. I also think that the next version of create react app will transpile dependencies, so this will be a non issue when that version arrives.

Thanks @timdorr for such a quick reply!
